### DISCUSSION TITLE
apex 23.2 built-in workflow engine

### DISCUSSION THREAD

**Discussion by j209** [flowsforapex/apex-flowsforapex#686] [General] [answered]
Hi
apex 23.2 has a built-in workflow engine.how about flow-for-apex and its feature?

========================================================
![photo_5812349069987790769_x](https://github.com/flowsforapex/apex-flowsforapex/assets/18129232/60f9ba5c-3ed9-4138-b8b9-e2ad16a7a56b)
![photo_5812349069987790770_x](https://github.com/flowsforapex/apex-flowsforapex/assets/18129232/7e48af3c-1e2c-47cb-afa1-3cc0f5f11f63)
![photo_6010489930125525782_y](https://github.com/flowsforapex/apex-flowsforapex/assets/18129232/759a9bef-b5f8-4002-8a47-786ba1da59c5)

**Comment by commi235** [comment, marked as answer]
Native Workflow in APEX isn't using BPMN 2.0 modelling, whereas Flows is BPMN 2.0 compliant.
The intention is to have both solutions Flows for APEX and the native workflow in APEX available in parallel.
In addition we are also trying to make sure both workflow engines integrate easily with each other.

Neither is a full replacement of the other and both solutions delivery a very unique set of features.
If you already have BPMN 2.0 knowledge it's probably a good start to lean onto Flows and incorporate the native workflow when it makes your life easier. ( Think of approval tasks you can already initiate from Flows. )
